tools/nl/embeddings/sheets/embeddings_us_filtered.csv

```diff
@@ -1244,3 +1244,4 @@ Number,Name,Id,Description Strings,PaLM Generated Alternatives,ChatGPT descripti
 ,,dc/topic/HomeValue,how much do houses cost in; value of homes; housing unit cost; how expensive are homes in,,,Home Value,,how much do houses cost in; value of homes; housing unit cost; how expensive are homes in;Home Value;,5
 ,,dc/topic/WetBulbTemperature,,,for this area what is the temperature at which the human body can't cool itself anymore; what is the wet bulb temperature; how high or low is the wet bulb temperature predicted to be; give me the degree of the wetbulb temperature; wet bulb temperature prediction range,Wet Bulb Temperature,,for this area what is the temperature at which the human body can't cool itself anymore; what is the wet bulb temperature; how high or low is the wet bulb temperature predicted to be; give me the degree of the wetbulb temperature; wet bulb temperature prediction range;Wet Bulb Temperature;,
 ,,dc/topic/Temperature,,,what's the temperature like; what's the climate here; tell me about the weather;,Temperature,,what's the temperature like; what's the climate here; tell me about the weather;;Temperature;,
+,,dc/235cmp60ywm46,number of lawyers; people working in law; how many legal service workers are there; population working in legal services; population of legal service workers,,,Population Working in Legal Services,,number of lawyers; people working in law; how many legal service workers are there; population working in legal services; population of legal service workers;Population Working in Legal Services;,
```

Comment:
"number of lawyers" may not be right because these are people working in legal services.?

(Another is dc/2w4xphkt6vs62 which is offices of lawyers, but I think legal services is probably broader)